sci-electronics/kicad-i18n/kicad-i18n-5.0.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
I'm not sure that we should create this package. What's wrong with kicad-4.0.7's approach? Also, according to `LINGUAS -> L10N` transition notes, we should install all localizations inconditionally. Language selection is only relevant when file size is significant.

Reply:
At the bottom of https://wiki.gentoo.org/wiki/Localization/Guide , there are 3 references:

* https://blogs.gentoo.org/mgorny/2016/05/16/how-linguas-are-thrice-wrong/
* https://archives.gentoo.org/gentoo-dev/message/a08ea09c2c8e534fd9bc1146703c66ff
* https://archives.gentoo.org/gentoo-dev/message/41e09d1ddc8b30abb9f9d21d205b7b82

Reply:
Okay, arguments were convincing. I made kicad-i18n install all locales as it's only 20MB.

Reply:
You missed a part of my remarks that I now realize was not 100% clear (sorry): by "4.0.7 approach", I meant "not a separate package". I don't understand the need for a separate package here. I find the way i18n is handled in 4.0.7 (in the same `kicad` package) better.

Reply:
Hum, nevermind that, I withdraw that remark. With the meta package, it makes sense as-is.

Reply:
Not only that, but also consider that installing i18n package is essentially just extract-copy (done by CMake). If it were to be bundled as previously you would need to recompile the whole binary just to get 20MB of locale files. This is exactly what we want to avoid as per @mgorny's blog post above ;)

Reply:
Yes, I agree, that's why I withdrew my remark. If it wasn't for `kicad-meta` and the whole set of other `kicad-*` packages, I think I'd advise against creating a new package for i18n because the recompile would only happen when changing the `i18n` flag which wouldn't happen *so* often.

But in this context, it makes sense.